theblues/charmstore.py

```diff
@@ -1,7 +1,7 @@
 import logging
 
 import requests
-from requests.exceptions import HTTPError
+from requests.exceptions import HTTPError, RequestException, Timeout
```

Comment:
Minor nitpick; since we're already doing it elsewhere, can you reformat this to 

```
from requests.exceptions import (
    HTTPError, 
    RequestException, 
    Timeout,
    )
```